independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/deploy-snapshots.tpl.qute.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy Snapshots
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  deploy-snapshot:
+    runs-on: ubuntu-latest
+    name: Deploy Snapshot artifacts
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: {java.version}
+          cache: 'maven'
+          server-id: 'ossrh'
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+
+      - name: Deploy Snapshot
+        run: |
+          mvn -B clean deploy -DperformRelease=true -Drelease
```

Comment:
Should we use `./mvnw` here?

Reply:
Not every extension has the Maven Wrapper installed, hence why I preferred to use the bundled Maven instead

Reply:
Hmmmm, but this is the Quarkiverse template to create new extensions so the extension is created with the wrapper? Or they are not?

Reply:
Just tested, they are not

Reply:
We should fix that but I agree this calls for a separate PR.